src/lib/api/user.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import gql from 'graphql-tag';
+
+/**
+ * Internal dependencies
+ */
+import API from 'lib/api';
+
+const QUERY_CURRENT_USER = gql`
+	query Me {
+		me {
+			id
+			displayName
+			isVIP
+			organizationRoles {
+				nodes {
+					organizationId
+					roleId
+				}
+			}
+		}
+	}
+`;
+
+export async function getCurrentUserInfo() {
+	const api = await API();
+
+	let response = {};
+	try {
+		response = await api.query( { query: QUERY_CURRENT_USER } );
+	} catch ( err ) {
+		throw err;
+	}
```

Comment:
Since we're not doing anything with the error (except re-throwing), we could omit the try/catch.

Reply:
Good idea; ff0a619